### ISSUE TITLE
Sass issues

### ISSUE THREAD

**Issue by rwwagner90** [Addepar/ember-table#973]
I am hitting a weird error where if I install ember-table, it breaks PostCSS, unless I install and configure postcss-scss. It would be ideal if we could opt in/out of sass and just get a plain CSS file from ember-table so errors do not occur.

**Comment by mixonic** [comment]
@rwwagner90 hm, on v5.0.0? On that release scss should only be a dev dependency.

What is the error?

**Comment by rwwagner90** [comment]
@mixonic yeah on 5.0.0. ember-cli-sass is a devDep, yeah, but my guess is because this is an scss file it causes issues https://github.com/Addepar/ember-table/blob/master/app/styles/ember-table/default.scss

When I install `ember-table` and run `ember s` I get an error that it can no longer find my CSS I am importing from my app. If I remove ember-table or change my styles/app.css to styles/app.scss and add configure PostCSS to work with scss, things work again.

I believe the issue here is ember-table still requires styles to be scss, even though it does not actually require a dependency on ember-cli-sass anymore.
